composer.json

```diff
@@ -40,7 +40,19 @@
     },
     "config": {
         "allow-plugins": {
-            "phpstan/extension-installer": true
+            "phpstan/extension-installer": true,
+            "symfony/flex": true
+        }
+    },
+    "extra": {
+        "symfony": {
+            "require": "7.0"
+        }
+    },
+    "scripts": {
+        "auto-scripts": {
+            "cache:clear": "symfony-cmd",
+            "assets:install %PUBLIC_DIR%": "symfony-cmd"
```

Comment:
These scripts are only necessary for projects, not for libraries like `PropelBundle`, I think?